src/commonMain/kotlin/exchange.dydx.abacus/calculator/SubaccountTransformer.kt

```diff
@@ -199,6 +199,29 @@ internal class SubaccountTransformer {
         }
     }
 
+    internal fun applyIsolatedMarginAdjustmentToWallet(
+        wallet: Map<String, Any>,
+        subaccountNumber: Int?,
+        delta: Map<String, Double>?,
+        parser: ParserProtocol,
+        period: String
+    ): Map<String, Any> {
+        return if (delta != null) {
```

Comment:
can we make `delta` here non-nullable? and move the nullability check out to the consumer?